src/cli.yml

```diff
@@ -49,6 +49,26 @@ args:
         long: texture3
         takes_value: true
         help: Path to 2D RGBA texture for iChannel3
+    - wrap0:
```

Comment:
Each of these entries should also specify `possible_values` so that clap can automatically inform the user if they don't select one of the four possible values. This will also automatically update the `--help` text.

`possible_values: ["clamp", "tile", "mirror", "border"]`

https://github.com/clap-rs/clap/blob/v3.1.1/examples/tutorial_builder/README.md#enumerated-values

Reply:
The formatting from the `--help` text is pretty bad:
```
USAGE:
    shadertoy [OPTIONS] [shader] [SUBCOMMAND]

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information

OPTIONS:
    -e, --example <example>      Run example shader from examples/ directory
    -H, --height <height>        Sets window height [default: 400]
        --texture0 <texture0>    Path to 2D RGBA texture for iChannel0
        --texture1 <texture1>    Path to 2D RGBA texture for iChannel1
        --texture2 <texture2>    Path to 2D RGBA texture for iChannel2
        --texture3 <texture3>    Path to 2D RGBA texture for iChannel3
    -W, --width <width>          Sets window width [default: 600]
        --wrap0 <wrap0>          Wrap mode for iChannel0 [default: clamp]  [possible values: clamp, tile, mirror,
                                 border]
        --wrap1 <wrap1>          Wrap mode for iChannel1 [default: clamp]  [possible values: clamp, tile, mirror,
                                 border]
        --wrap2 <wrap2>          Wrap mode for iChannel2 [default: clamp]  [possible values: clamp, tile, mirror,
                                 border]
        --wrap3 <wrap3>          Wrap mode for iChannel3 [default: clamp]  [possible values: clamp, tile, mirror,
                                 border]

ARGS:
    <shader>    Path to fragment shader

SUBCOMMANDS:
    get     Download shaders from shadertoy.com
    help    Prints this message or the help of the given subcommand(s)
```